### PR TITLE
List structure

### PR DESCRIPTION
**Pull request by Lucas-Erkana**
✅ Create an index.html file and write your HTML markup here
✅ Create an index.js file and set an array of some simple to do tasks (array of objects).
             description [string].
             completed [bool].
             index: [number].
✅ Write a function to iterate over the tasks array and populate an HTML list item element for each task.
✅ On page load render the dynamically created list of tasks in the dedicated placeholder. The list should appear in order of the index values for each task.
✅ Create a style.css and set rules for the To-Do List
✅ All your source files (index.html, index.js, and style.css) must be located in /src directory and your distribution files will be generated by webpack and served by webpack dev server from /dist folder.

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -1,11 +1,21 @@
...
 <body>
-    <h1>Hello webpack!</h1>
+  <header>
+  <h1><%= htmlWebpackPlugin.options.title %></h1>
+  </header>
+  <section>
+  <ul class="todo-list">
+    <li>Today's TO DO</li>
+    <li class='list-item'><input class='item-input' id="desc-input" placeholder='Add to your list ...'/></li>
+  </ul>
```
Comment (tugsem) [inline src/index.html:18]: - [ ] When I run npm start I should see the styled list that you have created dynamically as below: 
image
your index page seems like this: 
image

**Review by tugsem — CHANGES_REQUESTED** [review]
##  STATUS: REQUIRED CHANGES ♻️
Hi @Lucash2022 , 

Good job so far! 👏 
There are some issues that you still need to work on to go to the next project but you are almost there!

Alt Text
## Highlights 🚀 
✔️Github flow
✔️Descriptive pull request
✔️Linters setup
## Required Changes ♻️
- [ ] Please leave the microverse logo from your readme template. 
- [ ] Please add the right command to run the project locally into your readme. 
- [ ] Please add the correct tools and libraries that user must have to run the project locally into your prerequisites section.
## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.html (hunk 1 above)
Comment (Lucash2022) [reply src/index.html:18]: @tugsem 
Thank you so much for the review.

I have kindly fixed the above issue at [this link](https://github.com/Lucash2022/To_Do_List/commit/44c1c147705bff74452908e1aa903d214ed648cc)

**Review by danmainah — APPROVED** [review]
Hi @Lucash2022, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 🎉 🎉 

## Highlights
✔️ No linter errors
✔️ Github flow was correctly used
✔️  Good design

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**